themesh.html

```diff
@@ -4,189 +4,227 @@
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Alex-Xiang.jpg" alt="Alex">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Alex Xiang</p>
-              <p class="mesh-title">Developer</p>
-            </div>
-          </div>
-          <div class="grid-element">
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Anthony-Hinchliff.jpg" alt="Anthony">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Anthony Hinchliff</p>
-              <p class="mesh-title">Developer</p>
-            </div>
-          </div>
-          <div class="grid-element">
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Drew-Flaherty.jpg" alt="Drew">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Drew Flaherty</p>
-              <p class="mesh-title">Developer</p>
-            </div>
-          </div>
-          <div class="grid-element">
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Lorenzo-Fontoura.jpg" alt="Lorenzo">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Lorenzo Fontoura</p>
-              <p class="mesh-title">Developer</p>
-            </div>
-          </div>
-          <div class="grid-element">
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Mitchell-Westmoreland.jpg" alt="Mitch">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Mitchell Westmoreland</p>
-              <p class="mesh-title">QA Tester</p>
-            </div>
-          </div>
-          <div class="grid-element">
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Jasmin-Farhi.jpg" alt="Jasmin">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Jasmin Farhi</p>
-              <p class="mesh-title">QA Tester</p>
-            </div>
-          </div>
-          <div class="grid-element">
-            <figure class="circular-landscape">
-              <img src="./images/The-Mesh/Lucas-Cullen.jpg" alt="Lucas">
-            </figure>
-            <div class="div-text">
-              <p class="mesh-name">Lucas Cullen</p>
-              <p class="mesh-title">Visionary</p>
+    <section id="fh5co-services" data-section="services">
+      <div class="fh5co-services">
+        <div class="container">
+          <div class="row">
+            <!-- HERO SECTION PHILOSOPHY -->
+            <div class="col-md-12 section-heading text-center">
+              <h3 class="to-animate">
+                At DLTx we use a
+                <a href="https://www.holacracy.org/explore">holacracy</a> based
+                governance. In holacracy the organizational structure is rather
```

Comment:
Appears to be a redundancy in the text here. Remove the final sentence (or edit).

Reply:
Suggest this updated text:
"DLTx uses Holocratic governance mechanisms. In an [Holacracy](https://www.holacracy.org/explore), the organizational structure is flat, with limited hierarchy. Self-determined roles allows the mesh flexibility to take on varied tasks by choice, and move between Circles freely."